controllers/tenant/namespaces.go

```diff
@@ -73,6 +73,12 @@ func (r *Manager) syncNamespaceMetadata(ctx context.Context, namespace string, t
 				}
 			}
 
+			if tnt.Spec.Cordoned {
+				ns.Labels["projectcapsule.dev/cordoned"] = "true"
```

Comment:
offload the label to somewhere more central, so we have an overview what metadata we are using. I would suggest:

https://github.com/projectcapsule/capsule/blob/main/pkg/utils/tenant_labels.go

but in this case it can just be a constant